### PR TITLE
Style the button and tab view background to match the titlebar

### LINKED ISSUE #615: UI: align design with other UWP apps
It seems the design isn't following other Windows/UWP apps/components. The `+` button to add a tab seems very big and bold (it's even bigger than close/minimize window buttons).

Here is a comparison with Edge:

image

Also, it's positionned at the right edge of the window instead of being next to the last tab like in Edge.

### PR DESCRIPTION
**Pull request by cinnamon-msft**
## Summary of the Pull Request
Added additional styling to the title bar so it looks cohesive and tweaked the + button so it's not so large.

 
## PR Checklist
* [x] Closes #615 
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Detailed Description of the Pull Request / Additional comments
image

image

### CODE REVIEW

**Comment by DHowett-MSFT** [comment]
nit: rename pull request to be in imperative form. Imagine that you're completing the sentence,

"[if merged,] this pull request will..."

"style the title bar"

File: src/cascadia/TerminalApp/MinMaxCloseControl.xaml [hunk 1]
```diff
@@ -6,8 +6,8 @@
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d"
-    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
-    HorizontalAlignment="Left"
+		Background="{ThemeResource SystemChromeLowColor}"
+	  HorizontalAlignment="Left"
```
Comment (DHowett-MSFT) [inline src/cascadia/TerminalApp/MinMaxCloseControl.xaml]: spacing issues

File: src/cascadia/TerminalApp/MinMaxCloseControl.xaml [hunk 2]
```diff
@@ -130,7 +130,10 @@
     </StackPanel.Resources>
 
     <Grid x:Name="Content"></Grid>
-    <Border Height="36.0" MinWidth="160.0" x:Name="DragBar" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" DoubleTapped="DragBar_DoubleTapped"/>
+	<Border Height="36.0"
+					MinWidth="160.0"
```
Comment (DHowett-MSFT) [inline src/cascadia/TerminalApp/MinMaxCloseControl.xaml]: there's a loottt of spaces here. could you align the properties vertically after the space after the tag name?

File: src/cascadia/TerminalApp/MinMaxCloseControl.xaml [hunk 3]
```diff
@@ -31,7 +31,7 @@
                 </ResourceDictionary>
                 <ResourceDictionary x:Key="Dark">
                     <x:Double x:Key="CaptionButtonStrokeWidth">1.0</x:Double>
-                    <StaticResource x:Key="CaptionButtonBackground" ResourceKey="SystemControlBackgroundAltHighBrush"/>
+                    <StaticResource x:Key="CaptionButtonBackground" ResourceKey="SystemChromeLowColor"/>
```
Comment (DHowett-MSFT) [inline src/cascadia/TerminalApp/MinMaxCloseControl.xaml]: you might want to make these Transparent so the Row peeks through

File: src/cascadia/TerminalApp/MinMaxCloseControl.xaml [hunk 4]
```diff
@@ -18,7 +18,7 @@
             <ResourceDictionary.ThemeDictionaries>
                 <ResourceDictionary x:Key="Light">
                     <x:Double x:Key="CaptionButtonStrokeWidth">1.0</x:Double>
-                    <StaticResource x:Key="CaptionButtonBackground" ResourceKey="SystemControlBackgroundAltHighBrush"/>
+                    <StaticResource x:Key="CaptionButtonBackground" ResourceKey="SystemChromeLowColor"/>
```
Comment (DHowett-MSFT) [inline src/cascadia/TerminalApp/MinMaxCloseControl.xaml]: (as below)

File: src/cascadia/TerminalApp/TerminalPage.xaml [hunk 5]
```diff
@@ -1,45 +1,52 @@
...
-            <local:MinMaxCloseControl
-                x:Name="MinMaxCloseControl"
-                Grid.Column="3"
-                HorizontalAlignment="Right" />
-        </Grid>
-
-        <Grid x:Name="TabContent" Grid.Row="1" VerticalAlignment="Stretch" HorizontalAlignment="Stretch" />
-    </Grid>
-</Page>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
```
Comment (DHowett-MSFT) [inline src/cascadia/TerminalApp/TerminalPage.xaml]: !!! ALARM !!! this whole file is showing up as reformatted (!) what happened?

File: src/cascadia/TerminalApp/TerminalPage.xaml [hunk 6]
```diff
@@ -23,21 +23,28 @@ the MIT License. See LICENSE in the project root for license information. -->
                 <ColumnDefinition Width="Auto"/>
             </Grid.ColumnDefinitions>
 
-            <mux:TabView x:Name="TabView" Grid.Column="0" />
+            <mux:TabView x:Name="TabView" Grid.Column="0"
+									 SelectionChanged="TabView_SelectionChanged" />
```
Comment (DHowett-MSFT) [inline src/cascadia/TerminalApp/TerminalPage.xaml]: remove this and the other selectionchanged thingos

File: src/cascadia/TerminalApp/MinMaxCloseControl.xaml (hunk 2 above)
Comment (zadjii-msft) [reply src/cascadia/TerminalApp/MinMaxCloseControl.xaml]: (I think this is a tab/spaces thing)

File: src/cascadia/TerminalApp/MinMaxCloseControl.xaml [hunk 7]
```diff
@@ -130,7 +130,11 @@
     </StackPanel.Resources>
 
     <Grid x:Name="Content"></Grid>
-    <Border Height="36.0" MinWidth="160.0" x:Name="DragBar" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" DoubleTapped="DragBar_DoubleTapped"/>
+	<Border Height="36.0"
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/MinMaxCloseControl.xaml]: Why does GH think this line wrapped?

Comment (cinnamon-msft) [reply src/cascadia/TerminalApp/MinMaxCloseControl.xaml]: I added that in

**Comment by cinnamon-msft** [comment]
> I think I'm cool with this. I can definitely merge this into #1948 without too much pain.
> 
> Aside, does this fix #1913?

No I don't think so. When testing I was still experiencing the bug

**Comment by zadjii-msft** [comment]
@cinnamon-msft cool, cool cool cool cool cool.

**Comment by brainplot** [comment]
I'm running version 0.3.2171.0 freshly downloaded from the Microsoft Store. I've set my theme to _One Half Dark_ but the buttons and tabs remain white. Is there any additional settings I should add to my `profiles.json`?

terminal

**Comment by Stanzilla** [comment]
You need to switch your Windows theme from light to dark in the settings app.

**Comment by brainplot** [comment]
Oh, I understand now. I prefer a light theme for the rest of the system. I guess I'll stick with the white borders for now. It'd be nice to be able to change this from the `profiles.json` for the Terminal app only.

**Comment by mdtauk** [comment]
You can set the app to use dark themes in the settings

**Comment by DHowett-MSFT** [comment]
Yeah, it’s called “requestedTheme”.

**Comment by brainplot** [comment]
Thank you! I was indeed going to ask what the option is :)

**Pull request merged**